scripts/stop-server

```diff
@@ -1,12 +1,12 @@
 #!/bin/bash
-home=`dirname $0`/..
+home=$( dirname "$0" )/..
```

Comment:
`$(dirname $(dirname "$0"))` would make a cleaner pathspec. But since we actually want this to be the home directory, why not use `$HOME`?

Reply:
It's not `$HOME`, it's the root of the viroverse checkout, which in our production environment is  the vverse user's `$HOME/viroverse/` but needn't be. I can take your first suggestion though.